frontend/javascripts/oxalis/view/version_list.js

```diff
@@ -54,10 +54,24 @@ export async function previewVersion(versions?: Versions) {
   await api.tracing.restart(annotationType, annotationId, controlMode, versions);
   Store.dispatch(setAnnotationAllowUpdateAction(false));
 
-  const segmentationLayer = Model.getSegmentationTracingLayer();
-  const shouldPreviewVolumeVersion = versions != null && versions.volume != null;
-  const shouldPreviewNewestVersion = versions == null;
-  if (segmentationLayer != null && (shouldPreviewVolumeVersion || shouldPreviewNewestVersion)) {
+  if (versions == null) {
```

Comment:
Just a suggestion (untested):

```javascript
  const segmentationLayersToReload = []
  if (versions == null) {
    // No versions were passed which means that the newest annotation should be
    // shown. Therefore, reload all segmentation layers.
    segmentationLayersToReload.push(...Model.getSegmentationTracingLayers());
  } else if (versions.volumes != null) {
    // Since volume versions were specified, reload the volumeTracing layers
    const versionedSegmentationLayers = Object.keys(versions.volumes).map(volumeTracingId =>
      Model.getSegmentationTracingLayer(volumeTracingId)
    );
    segmentationLayersToReload.push(...versionedSegmentationLayers);
  }
  
  for (const segmentationLayer of segmentationLayersToReload) {
    segmentationLayer.cube.collectAllBuckets();
    segmentationLayer.layerRenderingManager.refresh();
  }
```

Reply:
Great, thank you!